src/components/app.tsx

```diff
@@ -263,21 +279,27 @@ export class App extends React.PureComponent<IProps, IState> {
     } else if (page >= 0 && (activity && page <= activity.pages.length)) {
       this.setState({currentPage: page, incompleteQuestions: []});
       setDocumentTitle(activity, page);
+      Logger.updateActivityPage(page);
+      Logger.log(LogEventName.change_activity_page, { new_page: page });
```

Comment:
Not sure if it's worth it, but are we following LARA log format? I guess LARA should emit something similar.

Reply:
I wasn't entirely sure on these new events.  I tried to follow the LARA structure (`event`, `parameters`, etc.) and then added new fields that were requested by the story (many of which are state values independent of the event - things like current activity, current page, etc.).  Hopefully my last commit makes this function call clearer so you can see what is actually going to be in the final log message.